org.flatpak.am-okay.yml

```
@@ -0,0 +1,81 @@
+id: org.flatpak.am-okay
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk 
+
+command: am-okay
+
+finish-args:
+    - --share=network
+    - --socket=x11
+    - --filesystem=host
+    - --filesystem=/tmp
+    - --device=all
+    - --filesystem=/media
+
+modules:
+    - name: am-okay-package
+      buildsystem: simple      
+      sources:
+          - type: git 
+            url: https://github.com/baldeuniversel/am-okay.git 
+            commit: e38467880012000a0d47aa363710f0fcd1e1538b 
+
+      build-commands: 
+            - install -D $FLATPAK_BUILDER_BUILDDIR/am-okay/bin/am-okay /app/bin/am-okay
+
+            - install -D $FLATPAK_BUILDER_BUILDDIR/am-okay/usr/share/metainfo/org.flatpak.am-okay.metainfo.xml \
+              /app/usr/share/metainfo
```

This install path is not correct. Use this instead:

```suggestion
              /app/share/metainfo
```

By the way, the other install paths below this one look unusual as well.

---

Done !